packages/firebase/README.md

```diff
@@ -36,16 +36,31 @@ you should use the
 
 ## Get the code (browser)
 
-### Script include
->This brings in all Firebase features. See
->["Include only the features you need"](#include-only-the-features-you-need)
->below for
->how to minimize download size by only including the scripts you need.
+We recommend only installing the features you need. The individually installable services are:
 
-Include Firebase in your web application via a `<script>` tag:
+- `firebase-app` - The core `firebase` client (required).
```

Comment:
misssing app-check

Reply:
Oops, added.

Reply:
Did you push it? I still don't see appcheck

Reply:
Missed error on push because I didn't pull the branch after committing the suggestion below.